examples/src/main/scala/org/apache/spark/examples/LocalALS.scala

```diff
@@ -95,7 +95,7 @@ object LocalALS {
 
   def showWarning() {
     System.err.println(
-      """WARN: This is a naive implementation of ALS and is given as an example!
+      s"""WARN: This is a naive implementation of ALS and is given as an example!
```

Comment:
please here revert the change and do the same in all similar places, since there is no variable to interpolate